virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY                                     
+[id="virt-hot-plugging-network-interfaces"]                            
+= Hot plugging secondary network interfaces                                            
+include::_attributes/common-attributes.adoc[]                  
+:context: virt-hot-plugging-network-interfaces                        
+                                                                
+toc::[]     
+
+You can add or remove secondary network interfaces without stopping your virtual machine (VM). {VirtProductName} supports hot plugging and hot unplugging for Linux bridge interfaces that use the VirtIO device driver.
+
+:FeatureName: Hot plugging and hot unplugging bridge network interfaces
+include::snippets/technology-preview.adoc[]
+
+[id="virtio-limitations_{context}"]
+== VirtIO limitations
+Each VirtIO interface uses one of the limited Peripheral Connect Interface (PCI) slots in the VM. There are a total of 32 slots available. The PCI slots are also used by other devices and must be reserved in advance, therefore slots might not be available on demand. {VirtProductName} reserves up to four slots to be used later for hot plugging interfaces. If your VM has four or more existing interfaces, you cannot hot plug any additional network interfaces. 
+
+[NOTE]
+====
+The actual number of slots available for hot plugging also depends on the machine type. For example, the default PCI topology for the q35 machine type supports hot plugging a single PCIe device, either emulated or assigned from the host. For more information on PCI topology and hot plug support, see the link:https://libvirt.org/pci-hotplug.html[libvirt documentation].
```

Comment:
`the default PCI topology for the q35 machine type supports hot plugging a single PCIe device` - this sentence gives the feeling that with q35 only one interface can be plugged, it is wrong.
I would change to "For example, the default PCI topology for the q35 machine type always adds one extra PCI slot".